build/cqs.tsv

```diff
@@ -45,6 +49,7 @@ com.thoughtworks.paranamer:paranamer	2.6	compile
 com.typesafe:config	1.2.1	compile
 com.typesafe.scala-logging:scala-logging_2.11	3.1.0	compile
 com.vividsolutions:jts	1.13	compile
+com.vlkan:flatbuffers	1.2.0-3f79e055	compile
```

Comment:
Are these definitely gonna be in the final version?  I'm wondering if we can queue up some CQs.